auth/http.go

```diff
@@ -1367,6 +1367,16 @@ func getIdentityGroupIDs(central *ImqsCentral, userId authaus.UserId) []authaus.
 	return []authaus.GroupIDU32{}
 }
 
+func filterGroups(groups []string, exclude string) []string {
+	var filtered []string
+	for _, group := range groups {
+		if group != exclude {
+			filtered = append(filtered, group)
```

Comment:
See this nice trick which is much faster (switch the if statement): authaus/auth.go:1094
`a = append(a[:i], a[i+1:]...)`